src/tables/duplicated_data.py

```diff
@@ -0,0 +1,20 @@
+from .table import Table
+
+class Duplicated_Data(Table):
```

Comment:
This whole table will not exist now. Our new instructions are to make duplicate data a flag and forget about identifying duplicates explicitly.

Reply:
This is on my list of things to take care of next sprint, along with other changes, since the new information came too late into the sprint to course-correct.